install/install.ps1

```diff
@@ -54,7 +54,10 @@ if (Test-Path -Path $latestDirectory) {
 
 # We use a directory junction here because not all Windows users will have permissions to create a symlink.
 Write-Verbose "Linking ${latestDirectory} to ${extractedDirectory}"
-$null = New-Item -Path $latestDirectory -Target $extractedDirectory -ItemType Junction
+cmd.exe /Q /C "mklink /J $latestDirectory $extractedDirectory" > $null
+if (!$?) {
+    Write-Error "Linking failed!"
+}

```

Comment:
Understanding _why_ we use `cmd.exe` here instead of just using PowerShell would probably be good.

Reply:
Same think.